src/Entity/DatasetSubmission.php

```diff
@@ -992,15 +992,6 @@ class DatasetSubmission extends Entity
      */
     protected $distributionPoints;
 
-    /**
-     * ERDDAPP Url for the dataset.
-     *
-     * @var string
-     *
-     * @ORM\Column(type="text", nullable=true)
-     */
-    protected $erddapUrl;
```

Comment:
I think there is a missing migration, and what do we want to do about the existing data in those fields?

Reply:
Yes, forgot to put that in.